compatibility/elementor/widgets/form.php

```diff
@@ -1743,7 +1765,7 @@ protected function register_controls() {
 				array(
 					'label'      => __( 'Fields width', 'jet-form-builder' ),
 					'type'       => Controls_Manager::SLIDER,
-					'size_units' => array( 'px', '%', 'em', 'rem', 'custom' ),
+					'size_units' => array( 'px', 'em', 'rem', 'custom' ),
```

Comment:
Please do not delete the unit percentage. Let you not use it, but for backward compatibility it should be saved